clinica/iotools/converters/adni_to_bids/adni_utils.py

```diff
@@ -37,8 +62,8 @@ def visits_to_timepoints(
     # We try to obtain the corresponding image Visit for a given VISCODE
     for adni_row in adnimerge_subj.iterrows():
         visit = adni_row[1]
-
-        preferred_visit_name = get_preferred_visit_name(visit)
+        study = ADNIStudy.from_string(visit.ORIGPROT)
```

Comment:
where does this ORIGPROT come from and what is it ? It doesn't look like it was there before

Reply:
nevermind I found it

Reply:
I agree with you it's a terrible name but I'm not the one to blame... ;)